csv2docx/cli.py

```diff
@@ -14,6 +14,14 @@
 @click.option(
     '--delimiter', '-d',
     default=";",
-    help='delimiter used in your csv. Default is \';\'')
-def main(data, template, delimiter):
-	c2d.convert(data, template, delimiter)
+    help='Delimiter used in your csv. Default is \';\'')
+@click.option(
+    '--name', '-n',
+    required=True,
+    help='Naming scheme for output files.')
+@click.option(
+    '--path', '-p',
+    default="output",
+    help='The location to store the files.')
+def main(data, template, name, delimiter, path):
+    csv2docx.convert(data, template, name, path, delimiter)
```

Comment:
Can you change 'main' to accept the same order of parameters as `convert`? e.g. switch delimiter and path

```py
def convert(data, template, name, path="output", delimiter=";"):
```

Reply:
Yeah I didn't noticed that